internal/testrunner/system/system.go

```diff
@@ -0,0 +1,44 @@
+package system
+
+import (
+	"os"
+	"path"
+
+	"github.com/pkg/errors"
+)
+
+// Run runs the system tests for the given package.
+func Run(packageRootPath string) error {
+	systemTestsPath, err := findSystemTestsPath(packageRootPath)
+	if err != nil {
+		return err
+	}
+
+	runner, err := newRunner(systemTestsPath)
+	if err != nil {
+		return errors.Wrap(err, "could not instantiate system tests runner")
+	}
+
+	if err := runner.run(); err != nil {
+		return errors.Wrap(err, "system tests failed")
+	}
+
+	return nil
+}
+
+func findSystemTestsPath(packageRootPath string) (string, error) {
+	systemTestsPath := path.Join(packageRootPath, "_dev", "test", "system")
```

Comment:
I'm not quite sure why is there `_dev` folder here. Is it temporarily for development purposes?

Reply:
My goal with the `_dev` folder is to create a space in the package structure where "development time" files can be kept, e.g. test-related files. This folder would not be included in the package build (i.e. the result of `elastic-package build`) and therefore would never make it into the package registry, etc.

In some ways the `_dev` folder is similar to the `_meta` folders we see in Beats, except I wanted to make it quite explicit via the name that "this space was for development only purposes". That said, I'm open to other names or ways of accomplishing the same goal.

I'm experimenting with this idea for now (hence the draft status of this PR) but if it turns out to be useful I plan to create  an issue in the elastic-spec repo to discuss the proposal of such a folder.

Reply:
> My goal with the _dev folder is to create a space in the package structure where "development time" files can be kept, e.g. test-related files. This folder would not be included in the package build (i.e. the result of elastic-package build) and therefore would never make it into the package registry, etc.

I think I already pushed some nginx test files to the package-storage. I assume we'll to clean them up.

I like your idea, although I thought about simply skipping such files/dirs (test/, Dockerfile, etc.), but there can be plenty of them. 

Do you think such directory should be also on the dataset level?

>I'm experimenting with this idea for now (hence the draft status of this PR) but if it turns out to be useful I plan to create an issue in the elastic-spec repo to discuss the proposal of such a folder.

Yes, please do.

Reply:
Made a proposal issue for the `_dev` folder in the package-spec repo for discussion: https://github.com/elastic/package-spec/issues/22.